command.go

```diff
@@ -922,15 +922,14 @@ func (c *Command) DebugFlags() {
 
 // Name returns the command's name: the first word in the use line.
 func (c *Command) Name() string {
-	if c.name != "" {
```

Comment:
I always do this way: if the field of struct is nil or blank, initialize it at first and then return it.